modules/ROOT/pages/add-policy-implementation-files-to-published-policy-definition.adoc

```diff
@@ -0,0 +1,25 @@
+= Add Policy Implementation Files to Published Policy Definition
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+When a policy definition asset is published in Exchange, the asset details page displays a *Pending* flag next to *Implementations* in the left navigation bar. The flag indicates that the policy implementation has not been added. Add the policy implementation files to to the policy definition to enable the policy to be available in runtime. The required implementation files are a binary file such as a WebAssembly (WASM) file for Flex Gateway or JAR for Mule 4 runtime and a YAML metadata file. 
```

Comment:
```suggestion
When a policy definition asset is published in Exchange, the asset details page displays a *Pending* flag next to *Implementations* in the navigation bar. The flag indicates that the policy implementation has not been added. Add the policy implementation files to the policy definition to make the policy available in runtime. The required implementation files are a binary file such as a WebAssembly (WASM) file for Flex Gateway or JAR file for Mule, and a YAML metadata file. 
```

Reply:
Do you need this sentence? I'm not sure it adds anything to the meaning of the test.

The flag indicates that the policy implementation has not been added.